test/integration/verify/controls/azure_network_watcher.rb

```diff
@@ -1,6 +1,6 @@
 resource_group = attribute('resource_group',       default: nil)
-nw             = attribute('network_watcher_name', default: nil).first
-nw_id          = attribute('network_watcher_id',   default: nil).first
+nw             = attribute('network_watcher_name', default: nil) # .first TODO uncomment (breaking)
```

Comment:
We spoke about this being broken. If you tear down your environment and rebuild I think you'll find that you'll need to add the `.first` back to each of these lines.